Description: Update various files to Debian versions
 There are a number of #defines and similar which change based on the
 specific versions of software being compiled against.  Update these to
 match the current Debian build environment.
Author: Stephen Frost <sfrost@snowman.net>
Last-Update: 2013-07-22

--- a/postgis/sqldefines.h
+++ b/postgis/sqldefines.h
@@ -10,7 +10,7 @@
 #define POSTGIS_GEOS_VERSION 34
 #define POSTGIS_PROJ_VERSION 48
 #define POSTGIS_LIB_VERSION '2.1.1'
-#define POSTGIS_LIBXML2_VERSION 2.9.0
+#define POSTGIS_LIBXML2_VERSION 2.9.1
 
 /*
  * Define the build date and the version number
--- a/doc/Makefile.comments
+++ b/doc/Makefile.comments
@@ -21,8 +21,8 @@
 DATA_built=postgis_comments.sql raster_comments.sql topology_comments.sql
 
 # PGXS information
-PG_CONFIG = /usr/local/pgsql/9.2/bin/pg_config 
-PGXS := /usr/local/pgsql/9.2/lib/pgxs/src/makefiles/pgxs.mk
+PG_CONFIG = /usr/lib/postgresql/9.3/bin/pg_config 
+PGXS := /usr/lib/postgresql/9.3/lib/pgxs/src/makefiles/pgxs.mk
 include $(PGXS)
 
 # PGXS override feature. The ability to allow PostGIS to install itself
